src/devices/lcec_deasda.c

```diff
@@ -148,6 +165,18 @@ static const lcec_pindesc_t slave_pins[] = {{HAL_FLOAT, HAL_OUT, offsetof(lcec_d
     {HAL_BIT, HAL_IN, offsetof(lcec_deasda_data_t, fault_reset), "%s.%s.%s.srv-fault-reset"},
     {HAL_BIT, HAL_IN, offsetof(lcec_deasda_data_t, halt), "%s.%s.%s.srv-halt"},
     {HAL_U32, HAL_OUT, offsetof(lcec_deasda_data_t, operation_mode), "%s.%s.%s.srv-operation-mode"},
+    {HAL_FLOAT, HAL_OUT, offsetof(lcec_deasda_data_t, torque), "%s.%s.%s.srv-torque-rel"}, // relative value (5) - hence current would be redundant
+    {HAL_BIT, HAL_OUT, offsetof(lcec_deasda_data_t, di_1), "%s.%s.%s.srv-di1"},
```

Comment:
I'd love to see din/dout pins use somewhat standard names.  Right now, here are the flavors being used across the tree:

- `din-X`: digital in/out boards in general, EpoCAT
- `srv-dinX`: EL7041, EL70x1
- `srv-X-dinX`: EL7342
- `din-<NAME>`: OMRG5

Personally, I'd prefer `din-X` for pins where numeric IDs make sense, or `din-<NAME>` where the pins have specific names on the device or in the documentation.  OMRG5 has `din-pot`, `din-pc`, `din-ext1`, `din-ext2`, `din-ext3`, `din-mon0`, `din-mon1`, `din-mon2`, `din-pcl`, `din-stop`, `din-bkir`, and a couple others.

Would `din-X` work for you here?

Reply:
Yeah, makes sense. I like din-x.
However, I'd suggest keeping this nomenclature to actual physical inputs (i.e. connected to the servo drive in explicit 1:1 wiring).
This means that all the other pins remain in their current naming state.